File: v2/lua/video_stream/main.lua
local json = require('cjson')
local hc = require('lib.httpclient').new()

local url = 'http://api-videostream-bj.fengkongcloud.com/v3/saas/anti_fraud/videostream'
local accessKey = '{ACCESS_KEY}'

local payload = json.encode({
  accessKey = accessKey,
  appId = 'default',
  imgType = 'POLITICS_PORN_AD',
  audioType = 'NONE',
  imgCallback = 'https://jsonplaceholder.typicode.com/posts/',
  data = {
    streamType = 'NORMAL',
    tokenId = '{UID}',
    url = 'https://jsonplaceholder.typicode.com/posts/',
  }
})

local res = hc:post(url, payload)
local data = json.decode(res['body'])
-- 通过 code 和 riskLevel 判断返回，具体请参考接口文档
print(res['body'])

File: v2/python/video_stream/main.py
import requests

url = 'http://api-videostream-bj.fengkongcloud.com/v3/saas/anti_fraud/videostream'
access_key = '{ACCESS_KEY}'
uid = '{UID}'
video_url = 'https://jsonplaceholder.typicode.com/posts/'

payload = {
    'accessKey': access_key,
    'appId': 'default',
    'imgType': 'POLITICS_PORN_AD',
    'audioType': 'NONE',
    'imgCallback': 'https://jsonplaceholder.typicode.com/posts/',
    'data': {
        'streamType': 'NORMAL',
        'tokenId': uid,
        'url': video_url,
    },
}

res = requests.post(url, json=payload)
# 通过 code 和 riskLevel 判断返回，具体请参考接口文档
if res:
    print(res.json())
